table/server/underdb/hive/pom.xml

```diff
@@ -37,6 +37,11 @@
       <artifactId>guava</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.hotels</groupId>
+      <artifactId>hcommon-hive-metastore</artifactId>
+      <version>1.4.2</version>
+    </dependency>
```

Comment:
I think this dependency is of questionable quality...

Reply:
what is our standard for including something as a dependency? @gpang